docs/integrations/sources/salesforce.md

```diff
@@ -87,6 +87,13 @@ The Salesforce connector retrieves deleted records from Salesforce. For the stre
 
```

Comment:
can you add a section that explains the differences between REST API and BULK API and the limitations when syncing from BULK API? some of the important points to highlight: 

- What differences between BULK & REST? 
- When do we use BULK or REST? 
- What are the limitations of the BULK API? e.g; which fields cannot be synced via REST?

Reply:
Also the context you mentioned in the sheet should be added: 

```
1. docs for the api quota updated 
2. """make it clear which types of streams are not queryable and why""" We get the info about streams from salesforce DESCRIBE resource (some internal logic is used there to make it queryable or not), see https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_describeGlobal.htm
 2.1. We also exclude some of the streams which are marked as "queryable" but the error is raised when we try to get them (list was obtained experimentally).

```

Reply:
docs updated